.babelrc

```diff
@@ -17,6 +17,8 @@
 },
 "plugins": [
   "@babel/plugin-proposal-class-properties",
-  "@babel/plugin-syntax-dynamic-import"
+  // "@babel/plugin-proposal-object-rest-spread",
```

Comment:
Please take off the comments on this file

Reply:
thanks.

Reply:
done